Bio/SeqIO/SffIO.py

```diff
@@ -643,155 +643,6 @@ def _sff_read_roche_index(handle):
 _valid_UAN_read_name = re.compile(r"^[a-zA-Z0-9]{14}$")
 
 
-def _sff_read_seq_record(
```

Comment:
As I recall I had this as a top level self contained function for ease of development and debugging - but not essential.

Reply:
We can clean this up when we look at the indexing code.